src/main.c

```diff
@@ -165,6 +166,7 @@ OSMesg gPIMesgBuf[32];
 OSMesgQueue gPIMesgQueue;
 
 s32 gGamestate = 0xFFFF;
+// it's in reality one s32 not two u16
```

Comment:
`// D_800DC510 is externed as an s32 in other files. D_800DC514 is only used in main.c, likely a developer mistake.`